sweep/sweeper.go

```diff
@@ -1728,6 +1723,61 @@ func (s *UtxoSweeper) handleBumpEventTxPublished(resp *bumpResp) error {
 	return nil
 }
 
+// handleBumpEventTxFatal handles the case where there's an unexpected error
+// when creating or publishing the sweeping tx. In this case, the tx will be
+// removed from the sweeper store and the inputs will be marked as `Failed`.
```

Comment:
Nit: maybe add something like this at the end `which will signal to the sweeper to not retry them in other sets again`?

Reply:
Done.